integration-tests/src/dca.rs

```diff
@@ -518,7 +518,7 @@ mod omnipool {
 				DAI,
 				amount_out,
 				Balance::MAX,
-				trade
+				BoundedVec::truncate_from(trade)
```

Comment:
Can `.try_into().unwrap()` be used here? I would prefer it over something truncating.